src/core/RecyclerListView.tsx

```diff
@@ -304,7 +300,7 @@ export default class RecyclerListView<P extends RecyclerListViewProps, S extends
 
     public getCurrentScrollOffset(): number {
         const viewabilityTracker = this._virtualRenderer.getViewabilityTracker();
-        return viewabilityTracker ? viewabilityTracker.getLastOffset() : 0;
+        return (viewabilityTracker ? viewabilityTracker.getLastOffset() : 0) + this.props.distanceFromWindow!;
```

Comment:
If think it should be 0 if viewability tracker is unavailable otherwise without doing anything there will be a non zero offset.

Reply:
Also, minimum offset for viewability tracker is zero. Thus this should only be attributed if lastOffset > distanceFromWindow